docs/dsl2.rst

```diff
@@ -383,20 +383,21 @@ A module script can define one or more parameters using the same syntax of a Nex
     }
 
 
-Then, parameters are inherited from the including context. For example::
+Parameters set before the include statement are inherited from the including context but parameters set later are not. For example::
 
     params.foo = 'Hola'
-    params.bar = 'Mundo'
 
     include {sayHello} from './some/module'
 
+    params.bar = 'Mundo'
+
     workflow {
```

Comment:
Even if possible, I think we should not promote this practice. I'd suggest adding a warning mentioning that params set after the import are not inherited.

Reply:
This is the whole point of this PR - alert users that they need to specify parameters before the import if they want them to have any impact. The marked line has no effect on the workflow and this is clearly stated in the sentence above the code example. Check the diff.

Reply:
@pditommaso I am not quite sure I understand what you're saying.

My point for adding these lines was "Don't just tell, show." In this case, show that setting the parameter after the import is not "possible" and hence it's also not a "practice". My discussions here in the PR notes, however did go further and might have caused some confusion.

Do you propose to delete the marked lines? If the code example should not be expanded to show the import inheritance explicitly I can remove it.

Reply:
Apology the late reply. I've added a note following your suggestion 17bbc757. Merging manually. Thanks for contributing.

Reply:
No worries, we live in a busy world. Happy to help.